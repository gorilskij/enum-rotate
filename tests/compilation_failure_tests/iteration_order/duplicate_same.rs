use derive_enum_rotate::EnumRotate;

#[derive(EnumRotate)]
#[iteration_order(A, B, C)]
#[iteration_order(A, B, C)]
enum Enum {
    A, B, C
}
