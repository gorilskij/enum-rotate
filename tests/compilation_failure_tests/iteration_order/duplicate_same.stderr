error: Duplicate "iteration_order" attribute, please specify at most one iteration order
 --> tests/compilation_failure_tests/iteration_order/duplicate_same.rs:5:1
  |
5 | #[iteration_order(A, B, C)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/compilation_failure_tests/iteration_order/duplicate_same.rs:8:2
  |
8 | }
  |  ^ consider adding a `main` function to `$DIR/tests/compilation_failure_tests/iteration_order/duplicate_same.rs`
